repository/src/main/java/org/apache/atlas/tasks/TaskRegistry.java

```diff
@@ -128,6 +127,36 @@ public List<AtlasTask> getInProgressTasks() {
         return ret;
     }
 
+    public List<AtlasTask> getInProgressTasksES() {
```

Comment:
Couple of things
1. use try catch on that particular line where expect an error to come. Don't put everything inside try catch block
2. If you are unable to fetch from ES. Log it as critical error, and fallback to Janusgraph

Reply:
Noted, making changes for it